node/src/main/scala/coop/rchain/node/BlockStoreMigrate.scala

```diff
@@ -0,0 +1,66 @@
+package coop.rchain.node
+
+import coop.rchain.blockstorage.{FileLMDBIndexBlockStore, KeyValueBlockStore}
+import coop.rchain.casper.storage.RNodeKeyValueStoreManager
+import coop.rchain.metrics.Metrics
+import coop.rchain.rspace.Context
+import coop.rchain.shared.Log
+import coop.rchain.shared.syntax.sharedSyntaxKeyValueStoreManager
+import monix.eval.Task
+import org.rogach.scallop.ScallopConf
+import monix.execution.Scheduler.Implicits.global
+
+import java.nio.file.Path
+
+final case class Options(arguments: Seq[String]) extends ScallopConf(arguments) {
+  val width = 120
+  helpWidth(width)
+  printedName = "migrate-blockstore"
+
+  val originalBlockStore = opt[Path](
+    descr = "original block store directory",
+    required = true
+  )
+
+  val targetDataDir = opt[Path](
+    descr = "Output block store directory",
+    required = true
+  )
+
+  verify()
+
+}
+object BlockStoreMigrate {
+  def main(args: Array[String]): Unit = {
+    val options            = Options(args)
+    val originalBlockStore = options.originalBlockStore()
+    val targetDataDir      = options.targetDataDir()
+    implicit val log       = effects.log
+    implicit val metrics   = new Metrics.MetricsNOP[Task]
+
+    val t = for {
+      casperStoreManager <- RNodeKeyValueStoreManager[Task](targetDataDir)
+      blockKVStore       <- KeyValueBlockStore[Task](casperStoreManager)
+
+      blockFileStorage <- {
```

Comment:
The logic you have here can be inside function on companion object on FileBS. And this function can be called here with instance of FileBS and KVBS.
When we remove FileBS this migration function will not be needed anymore.

Reply:
Then this file will be deleted also.

Reply:
Makes sense. I was already start thinking about adding additional commands to help managing of export/import data from the databases.
For this migration it would be something like this.
`rnode dev migrateFileBlockStore [--source <fileBS-path>] [--target <kvBS-path>]`.

It's outside of the scope for this PR.